scripts/main.py

```
@@ -1,6 +1,45 @@
+    if value is None or value == "null":
+        return '-'
+    return f'{float(value):.2f}%'
+
+# Helper function to format NAV with two decimal places without a percentage symbol
+def format_nav(nav):
+    if nav is None or nav == "null":
+        return '-'
+    return f'{float(nav):.2f}'
+
+# Convert date from mm/dd/yyyy to dd-mm-yyyy format
+import os
```

Ruff (0.14.5)

Redefinition of unused `os` from line 1

Remove definition: `os`

(F811)